citi-onboarding/Delta-odonto
# Feature/front who we are

## What i did
adjust the width of the section who we are and the navbar. Adjust the color of the navbar

## How to test

- In the folder Delta-odonto run the command, check if you are in the branche develop, if you are not switch to develop with the command
`git checkout devlop`

- In the branch develop run the command
`git pull`

- Commit the changes

- Now, go to branch feature/front-whoWeAre with the command
`git checkout feature/front-whoWeAre`

-  Now you need to update this branch with the command
`git push origin develop`

- So you can see the section who whe are in your right position
